src/nvim/main.c

```diff
@@ -1396,6 +1396,24 @@ static void load_plugins(void)
 }
 
 /*
+ * Read all the early plugin files (those are sourced before the vimrc)
+ */
+static void load_early_plugins(mparm_T *parmp)
+{
+  if (parmp->use_vimrc != NULL) {
+    if (strcmp(parmp->use_vimrc, "NONE") == 0
+        || strcmp(parmp->use_vimrc, "NORC") == 0) {
+      if (parmp->use_vimrc[2] == 'N')
```

Comment:
This can't be true if `strcmp(parmp->use_vimrc, "NORC") == 0`.

> When {vimrc} is equal to "NORC" (all uppercase), this has the
same effect as "NONE", but loading plugins is not skipped.

Maybe we just don't need to check for `NORC`.